src/electron/windows/sign_windows_executable.action.mjs

```diff
@@ -192,7 +161,7 @@ async function signWindowsExecutable(exeFile, algorithm, options) {
   try {
     exitCode = await jsign(exeFile, jsignArgs);
```

Comment:
Check the exit code too.

Reply:
already checked, see the following lines.

Reply:
Ah, ok, It was collapsed.